githublib.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import requests
+import json
+import time
+from datetime import datetime
+
+from common import get_envvar
+
+SECONDS_PER_WEEK = 604800
+
+
+class Github:
+    def __init__(self, token, owner, repo, expected_label):
+        self.owner = owner
+        self.repo = repo
+        self.expected_label = expected_label
+        self.query_url = f"https://api.github.com/repos/{owner}/{repo}/issues"
+        self.headers = {"Authorization": f"token {token}"}
+
+    def get_issues(self, page):
+        params = {"state": "open", "page": page, "per_page": "100"}
+
+        r = requests.get(self.query_url, headers=self.headers, params=params)
+        data = r.json()
+        return data
+
+    def check_labels(self, issue):
+        if self.expected_label == "":
+            return True
+        for label in issue["labels"]:
+            if label["name"] == self.expected_label:
+                return True
+        return False
+
+    def check_time(self, issue, max_delta):
```

Comment:
What is this suppose to do? I do not understand.
Is it the issue aging time?

Reply:
we are reading x weeks back
https://github.com/oshoval/github2jira/pull/1/files#diff-63469d0fb919a44c962153d5288c638b1e9e5184a30d002a58eeb8a269e9878aR13

We don't read all the issues, because we don't want to open tickets for too old issues.
On the other hand an issue might be opened, and after just one week assigned a sig/network label, so its better to do scan the history, but for limited time.

Reply:
Same as the label then, this is something internal. And you need to give it a good name, because the current one says nothing meaningful.